tests/test_text/test_postag.py

```diff
@@ -2,13 +2,13 @@
 # tests.test_text.test_postag
 # Tests for the part-of-speech tagging visualization
 #
-# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Author:   Rebecca Bilbro
 # Created:  2017-03-22 15:46
 #
-# Copyright (C) 2017 District Data Labs
+# Copyright (C) 2018 The scikit-yb developers
 # For license information, see LICENSE.txt
 #
-# ID: test_postag.py [bd9cbb9] rebecca.bilbro@bytecubed.com $
+# ID: test_postag.py [bd9cbb9] $
```

Comment:
We should probably discuss this in #509 - the script I was referring to automatically creates this `ID` line and includes the email of the git user that made the first commit of the file. If you don't want those emails here, we'll have to adapt the script.

Reply:
Yes, I don't have a clear sense of what specifically #509 is going to do (e.g. what is manual and what is automatic). Perhaps some people would prefer not to share their emails?

Reply:
The emails are already on the git commit and can be seen by `git log` but you're right that we're making it a lot more prominent by putting it in the header of the files. When we get to #509 I'll make sure I ping you to carry on this discussion.